change/@fluentui-react-drawer-6bf581dd-1ead-41a7-8f5c-1e543da04967.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32818&commitId=873a3a513b25fb80affcd7e20e3a4263e2072b60&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|12| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1150| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>